addon/components/base/bs-dropdown/button.js

```diff
@@ -13,27 +13,15 @@ import { computed } from '@ember/object';
  @extends Components.Button
  @public
  */
-@classNames('dropdown-toggle')
-@attributeBindings('aria-expanded')
+@templateLayout(layout)
 export default class DropdownButton extends Button {
-  keyDown(e) {
+  @action
+  handleKeyDown(e) {
     this.get('onKeyDown')(e);
   }
 
   @computed('isOpen')
   get 'aria-expanded'() {
```

Comment:
```suggestion
   get ariaExpanded() {
```

Not related to this change but noticed when reviewing the template.